lblod/app-digitaal-loket
# [ Maintenance ] persons-sensitive deltas

# Description

DL-5492

This PR extends the configuration of the maintainer and job-initiator for person-sensitive deltas. It moves in `config.json` files environment variables of the docker-compose related to the services. It also removes the custom rule file to include it in the maintainer rule instead.

# Type of change


# Related services

- delta-publication-graph-maintainer
- delta-background-jobs-initiator

# How to test 

1. drc up -d loket + drc up -d loket-data-warehouse
2. Few things to tweak to trigger delta generation 
a. Remove additionalFilter in export.json from `config/delta-producer/persons-sensitive`
b. If you start with no initial sync on the producer side (loket) you'll have to provide in the consumer `DCR_START_FROM_DELTA_TIMESTAMP` you can set without time but it has to be before the current date where you generate new persons.
c. Add a matching key in the config.json file of your delta-producer-publication-graph-maintainer **Note that key had to be added in the file separately and cannot work with docker-compose.override.yml even if the variable is set**
4. Create a new person in loket (A delta file should be generated from here to be consumed in warehouse)

# What to check

- .N/A

# Links to other PR's

- N/A

# Notes
